llx/rawdata_json.go

```diff
@@ -343,9 +343,10 @@ func rawDataJSON(typ types.Type, data interface{}, codeID string, bundle *CodeBu
 			return nil
 		}
 
-		// if *time == NeverPastTime || *time == NeverFutureTime {
-		// 	TODO: ... unclear
-		// }
+		if *time == NeverPastTime || *time == NeverFutureTime {
+			buf.WriteString("Never")
```

Comment:
doesn't this need to be a valid json value?

Reply:
bleeeehhhh to json! 
updated :) 
thank you!